Sources/BitWriter.swift

```diff
@@ -25,6 +25,10 @@ public protocol BitWriter {
 
     /// Writes `number`, using and advancing by `bitsCount` BIT positions.
     func write(number: Int, bitsCount: Int)
+    
+    /// because you cant convert a UInt64 into an Int without potential for overflowing it, we need an unsingned variant of `write`
```

Comment:
I don't think this comment should be marked as documentation comment. Comments starting with three slashes are documentation comments. They are parsed and published as API documentation at [](https://tsolomko.github.io/BitByteData/).